packages/ui-shared/src/Dialog/index.tsx

```diff
@@ -24,13 +24,14 @@ export const Wrapper: FC<DialogProps> = props => {
     <Dialog
       className={style.dialog}
       transitionDuration={0}
-      canOutsideClickClose={false}
+      canOutsideClickClose
```

Comment:
This actually is a regression. We purposely disabled clicking outside dialogs to close them since you may click accidentaly outside while filling a form (or writing a cntent element, for example) which made you lose your current work. 

So escape to close is good, clicking outside should not close it

Reply:
I reverted the canOutsideClickClose, and i'll see what I can do for the width of the modal